README.md

```diff
@@ -55,10 +55,10 @@ Currently Nydus is supporting the following platforms in container ecosystem:
 | Storage/Build | [Harbor](https://github.com/goharbor/acceleration-service)                                                      | Provides a general service for Harbor to support acceleration image conversion based on kinds of accelerator like Nydus and eStargz etc                      | ✅      |
 | Distribution  | [Dragonfly](https://github.com/dragonflyoss/Dragonfly2)                                                         | Improve the runtime performance of Nydus image even further with the Dragonfly P2P data distribution system                                                  | ✅      |
 | Build         | [Buildkit](https://github.com/moby/buildkit/blob/master/docs/nydus.md)                                          | Provides the ability to build and export Nydus images directly from Dockerfile                                                                               | ✅      |
-| Runtime       | Kubernetes                                                                                                      | Run Nydus image using CRI interface                                                                                                                          | ✅      |
+| Runtime       | [Kubernetes](https://github.com/containerd/nydus-snapshotter/blob/main/docs/run_nydus_in_kubernetes.md)         | Run Nydus image using CRI interface                                                                                                                          | ✅      |
 | Runtime       | [Containerd](https://github.com/containerd/nydus-snapshotter)                                                   | Nydus Snapshotter, a containerd remote plugin to run Nydus image                                                                                             | ✅      |
 | Runtime       | [CRI-O / Podman](https://github.com/containers/nydus-storage-plugin)                                            | Run Nydus image with CRI-O or Podman                                                                                                                         | 🚧      |
-| Runtime       | [Docker](https://github.com/nydusaccelerator/docker-nydus-graphdriver)                                          | [Experimental] Run Nydus image in Docker container with graphdriver plugin                                                                                   | 🚧      |
+| Runtime       | [Docker / Moby](https://github.com/dragonflyoss/image-service/blob/master/docs/docker-env-setup.md)             | [Experimental] Run Nydus image in Docker container with graphdriver plugin or snapshotter                                                                    | 🚧      |
```

Comment:
please drop `graphdriver plugin` here and `[Experimental]`.

Run Nydus image in Docker container with containerd and nydus-snapshotter.

Reply:
`graphdriver plugin` is now useless, we could just list it as a sub-project, rather than highlight it here.

Reply:
ok